DEVELOPER_GUIDE.md

```diff
@@ -164,7 +164,21 @@ Run tests :
 ```
 ./gradlew clean build test
 ```
+## Generate Artifact
 
+In opensearch-sdk navigate to build/distributions. Look for tar ball in the form `opensearch-sdk-1.0.0-SNAPSHOT.tar`. If not found follow the below steps to create one:
+
+```
+./gradlew clean && ./gradlew build
+```
+Once the tar ball is generated navigate to /src/test/resources and look for extension.yml. Create one if not present
+
+Look for tar ball in /build/distributions. To run the artifact i.e. tar ball, run the below command
+
+```
+tar -xvf opensearch-sdk-1.0.0-SNAPSHOT.tar
+```
+For now, create the extension.yml file within  opensearch-sdk/build/distributions/opensearch-sdk-1.0.0-SNAPSHOT/src/test/resources/  and then run ./bin/opensearch-sdk to start opensearch
```

Comment:
Move this above [here](https://github.com/opensearch-project/opensearch-sdk/pull/84/files#diff-d5555724c2ebca1b1f9be92a74c90e84c2dcbc76631148801024e24372b7edc6R174)

Reply:
Not sure if the above link works. I meant here
```
Create one if not present (For now, create the extension.yml file within  opensearch-sdk/build/distributions/opensearch-sdk-1.0.0-SNAPSHOT/src/test/resources/  and then run ./bin/opensearch-sdk to start opensearch)....

Reply:
Once the user untar the artifact then this step has to be performed, therefore i have written it after that step

Reply:
Also, let's change it to
```
 For now until https://github.com/opensearch-project/opensearch-sdk/issues/86 is resolved

Reply:
Let's add details of what should be present in extensions.yml file

Reply:
done